dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/statistics/CpuUsageListener.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.rpc.cluster.loadbalance.statistics;
+
+public interface CpuUsageListener {
+    void cpuChanged(String ip, Float cpu);
```

Comment:
Consider a scenario that two dubbo application running on different nodes but share  same IP( NAT ). 
Which cpu usage info will be used  here?  How to deal with this problem ?

Reply:
Good catch, thanks. I think it should use IP and the port to avoid this.

Reply:
@ralf0131 What do you think?

Reply:
Hi, 

ip + port should be enough for this case.